rpmio/rpmkeyring.cc

```diff
@@ -61,6 +68,53 @@ rpmKeyring rpmKeyringFree(rpmKeyring keyring)
     return NULL;
 }
 
+rpmKeyringIterator rpmKeyringInitIterator(rpmKeyring keyring, int unused)
+{
+    if (!keyring || unused != 0)
```

Comment:
BTW kudos for this - we have way too many APIs that don't check their arguments.

Reply:
I am actually more proud I figured out this needs a forced value or it won't be able to be used in the future.

Reply:
That's what I mean. We have the NULLs all over the place but other arguments like the flags, not.